content/webapp/views/pages/concepts/concept/concept.RelatedConceptsGroup.tsx

```diff
@@ -45,10 +46,16 @@ const RelatedConceptsGroup: FunctionComponent<Props> = ({
   buttonColors,
   dataGtmTriggerName,
 }: Props) => {
+  const { config } = useConceptPageContext();
+
   if (!relatedConcepts || relatedConcepts.length === 0) {
     return null;
   }
 
+  const displayRelatedConcepts = relatedConcepts.filter(
```

Comment:
I don't know to what extent this is necessary or if this is already the related concepts that we expect to get from the API. I didn't see any difference between `relatedConcepts` and `displayRelatedConcepts` in my (minimal) testing